tests/bip39.rs

```diff
@@ -66,6 +66,47 @@ fn test_wordlist_codec() {
     }
 }
 
+#[test]
+fn test_mnemonic_phrase_when_separator_is_repeated() {
+    let test_cases = &[
+        // U+3000 separator
+        ("　", true),
+        // whitespace(U+0020) is also allowed as a separator, because U+3000 is normalized to the whitespace
+        (" 　", false),
+        (" ", true),
+        ("  ", false),
+        ("　 ", false),
+        ("　 　", false),
+    ];
+
+    for case in test_cases {
+        let mnemonic_phrase = format!("あいこくしん{}あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あいこくしん　あおぞら", case.0);
+        if case.1 {
```

Comment:
Can you not just assert_eq the Result is_ok with case.1 instead of doing two times the same block?

Reply:
right, fixed